cmd/otelcorecol/go.mod

```diff
@@ -95,7 +95,7 @@ require (
 	go.opentelemetry.io/collector/connector/connectorprofiles v0.111.0 // indirect
 	go.opentelemetry.io/collector/connector/connectortest v0.0.0-20241021181817-007f06b7c4a8 // indirect
 	go.opentelemetry.io/collector/consumer v0.111.1-0.20241021181817-007f06b7c4a8 // indirect
-	go.opentelemetry.io/collector/consumer/consumererror v0.0.0-20241021181817-007f06b7c4a8 // indirect
+	go.opentelemetry.io/collector/consumer/consumererror v0.111.0 // indirect
```

Comment:
I don't think I should touch this file, but `make gotidy` was modifying it and all tests were failing without it.